go-controller/pkg/ovn/egressip.go

```diff
@@ -1857,6 +1863,12 @@ type egressIPController struct {
 // (routing pod traffic to the egress node) and NAT objects on the egress node
 // (SNAT-ing to the egress IP).
 func (e *egressIPController) addPodEgressIPAssignment(egressIPName string, status egressipv1.EgressIPStatusItem, pod *kapi.Pod, podIPs []*net.IPNet) (err error) {
+	start := time.Now()
+	defer func() {
```

Comment:
I think this should be a level 5 log output. I may not understand why we need to record the latency.

Reply:
Talked some off offline. We will keep this as a metric, but will disable it by default, as it is only expected to be useful for egress scaling tests.

Reply:
@martinkennelly please help with the flags here! I wonder if you think we could piggy back on `ovnkube_config_duration_enable` or create a new one

Reply:
Disabled by default. To enable this metric, use newly added flag: `--metrics-enable-eip-scale`